src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $totalSize + $totalGap + 10px;
+  padding: 10px;
+
+  $days: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
+
+  @each $day, $number in $days {
+    &--start-day--#{$day} .calendar__day:first-child {
+      margin-left: ($number * ($daySize + $dayGap)) + 3 * $dayGap;
```

Comment:
You don't need use `3 * $dayGap` at the end. If you need this there is some mistake with your width calculation. Check it, please